### PR TITLE
[full-ci] Upload progress & overlay improvements

### PR DESCRIPTION
**Pull request by JammingBen**
## Description

* Remove fetching of newly uploaded files to improve performance
* Redesign the upload overlay
* Show currently uploading files in the upload overlay
* Immediately show the upload overlay when uploading folders to tell the user that the upload is starting 
* Only show top level folders in the upload overlay when uploading folders
* Remove the Uppy StatusBar plugin

## Images

image

image

image

image

image

## Related Issue

- Works towards https://github.com/owncloud/web/issues/7066
- Fixes https://github.com/owncloud/web/issues/7068

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

### CODE REVIEW

**Review by pascalwengerter — COMMENTED** [review]
Love it! Added small UI/markup suggestions, seems to work like a charm (and I've restarted CI which failed with a middleware-startup-timeout)

File: packages/web-runtime/src/components/UploadInfo.vue [hunk 1]
```diff
@@ -1,32 +1,104 @@
...
-          <oc-icon name="check" variation="success" size="small" />
+          <oc-icon v-if="item.status === 'error'" name="close" variation="danger" size="small" />
+          <oc-icon
+            v-else-if="item.status === 'success'"
+            name="check"
+            variation="success"
+            size="small"
+          />
+          <oc-icon v-else-if="item.status === 'cancelled'" name="close" size="small" />
+          <oc-icon v-else-if="uploadsPaused" name="pause" size="small" />
```
[the remark this answers is not in the document]
Comment (JammingBen) [reply packages/web-runtime/src/components/UploadInfo.vue:95]: @tbsbdr What do you think? We could keep displaying the loading wheel during pause.

**Review by pascalwengerter — COMMENTED** [review]
Should have checked this earlier sry, just found this glitch on mobile 😬

File: packages/web-runtime/src/components/UploadInfo.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (kulmann) [reply packages/web-runtime/src/components/UploadInfo.vue:96]: unrelated: the folder icon looks rather big here 😅

**Review by pascalwengerter — APPROVED** [review]
😻

**Pull request merged**